mlflow/tensorflow/keras.py

```diff
@@ -280,7 +412,7 @@ def _is_tf_keras(model):
     # append loader_module, data and env data to mlflow_model
     pyfunc.add_to_model(
         mlflow_model,
-        loader_module="mlflow.keras",
+        loader_module="mlflow.tensorflow.keras",
```

Comment:
```suggestion
        loader_module="mlflow.tensorflow",
```

Can we use `mlflow.tensorflow` as a loader module?

Reply:
we can. I will update this.